js/lms-ui-multiselect.js

```diff
@@ -278,6 +278,7 @@ function multiselect(options) {
 			var exclusive = $(this).attr('data-exclusive');
 			var selected = $(this).is(':selected');
 			var disabled = $(this).is(':disabled');
+			var blended = $(this).attr('data-blended');
 			var class_name = 'visible' + (exclusive === '' ? ' exclusive' : '');
```

Comment:
blended ma mieć inne znaczenie niż disabled? Co dokładnie ma oznaczać blended?
Nie będzie wizualnie kolidować z disabled?

Reply:
'blended' ma znaczyć, że opcja jest aktywna do zaznaczenia, ale tylko tekst opcji jest 'na szaro'. Generalnie coś takiego jak używana klasa 'blend'. Wizualnie kolor jest ten sam to może i będzie lekka kolizja, zatem może dać klasę 'teal' wtedy tekst opcji będzie taki niebieskawy ?
![Zrzut ekranu z 2021-01-14 20-23-50](https://user-images.githubusercontent.com/7595420/104638434-73113700-56a6-11eb-870a-905761e0e3ee.png)

Reply:
jakby co to dodałem poprawkę z przekreśleniem :)

Reply:
Jeszcze zapytam z ciekawości: czy styl przekreślenia w CSS jest regulowalny? Można by jakoś bardziej dyskretnie kreślić (zdaję sobie sprawę, że w CSS może nie być takiej możliwości ;-)).

Reply:
można zmniejszyć krycie koloru linii np. .crossed	`{ text-decoration: line-through; text-decoration-color: rgba(0, 0, 0, 0.5);}`
wtedy przekreślenie jest bardziej subtelne

Przed:
![Zrzut ekranu z 2021-01-15 11-37-00](https://user-images.githubusercontent.com/7595420/104714867-050c5480-5726-11eb-9e7c-448695792d54.png)

Po:
![Zrzut ekranu z 2021-01-15 11-42-21](https://user-images.githubusercontent.com/7595420/104715367-c2974780-5726-11eb-9bdd-9911e17f2877.png)

Jakaś subtelna różnica jest, ale ktoś zaraz powie, że jak ma słaby wzrok to nie widzi przekreślenia.

Reply:
Moim zdaniem domyślne przekreślenie zamazuje trochę oryginalną treść - lepiej właśnie subtelnie!
Podobnie można by poprawić dla zobowiązań klienta, które nie są jeszcze zatwierdzone ;-)

Reply:
No to proponuje jeszcze coś takiego:
![Zrzut ekranu z 2021-01-15 13-10-12](https://user-images.githubusercontent.com/7595420/104725941-190a8300-5733-11eb-8d6f-58eff1f8cb55.png)